### PR TITLE
Fixed conversion warnings. Step 7

### PR DESCRIPTION
**Pull request by igorkorsukov**

### CODE REVIEW

File: src/engraving/infrastructure/internal/engravingconfiguration.cpp [hunk 1]
```diff
@@ -149,22 +149,22 @@ double EngravingConfiguration::guiScaling() const
     return uiConfiguration()->guiScaling();
 }
 
-Color EngravingConfiguration::selectionColor(int voice) const
+Color EngravingConfiguration::selectionColor(engraving::voice_idx_t voice) const
```
Comment (Jojo-Schmitz) [inline src/engraving/infrastructure/internal/engravingconfiguration.cpp:152]: why that `engraving::` here but not further down?

File: src/engraving/layout/layoutpage.cpp [hunk 2]
```diff
@@ -403,10 +403,10 @@ void LayoutPage::layoutPage(const LayoutContext& ctx, Page* page, qreal restHeig
         qreal ndist = si->distance() - si->height();        // next taller system
         qreal fill  = ndist - dist;                         // amount by which this system distance exceeds next shorter
         if (fill > 0.0) {
-            qreal totalFill = fill * i;                     // space required to add this amount to all shorter systems
+            qreal totalFill = fill * static_cast<qreal>(i);                     // space required to add this amount to all shorter systems
```
Comment (Jojo-Schmitz) [inline src/engraving/layout/layoutpage.cpp:406]: Better reduce the amount of spaces, to keep the comment aligned

File: src/engraving/infrastructure/internal/engravingconfiguration.cpp (hunk 1 above)
Comment (igorkorsukov) [reply src/engraving/infrastructure/internal/engravingconfiguration.cpp]: removed

File: src/engraving/layout/layoutpage.cpp (hunk 2 above)
Comment (igorkorsukov) [reply src/engraving/layout/layoutpage.cpp]: done, thanks

**Review by handrok — APPROVED** [review]

**Comment by igorkorsukov** [comment]
@Jojo-Schmitz This is the last series of my warning fixes, there are a still warnings, it will be great if you fix it :)

**Comment by Jojo-Schmitz** [comment]
Will do, watch out for an updated #11458 shortly

**Comment by Jojo-Schmitz** [comment]
> @Jojo-Schmitz This is the last series of my warning fixes, there are a still warnings, it will be great if you fix it :)

Done, see #11458

**Pull request merged**